components/ui/ContextNavBar.css

```diff
@@ -20,7 +20,7 @@
 .navbar-context,
 .navbar-context .navbar-inner {
   background-color: #fff;
-  z-index: 420;
+  z-index: var(--contextbarZIndex);
```

Comment:
I was hesitant to not remove the line with `z-index: 420;` and keep both, like this:
```css
  z-index: 420;
  z-index: var(--contextbarZIndex);
```
So that is would not be a breaking change for SQ, wdyt ?

Reply:
It would probably be confusing... We'd at least need a comment in that case. But I don't think it's necessary. The Z-index remains the same value, after all.

Reply:
Yeah indeed, so you'll have to think about adding this value to your theme file when you update to this version of SUC.